src/ewatercycle_wflowjl/model.py

```diff
@@ -201,39 +200,13 @@ def _make_cfg_file(self, **kwargs) -> Path:
         return config_file
 
 
-def install_wflow():
-    """Install Wflow.jl with the BMI branch."""
-    jl.seval("import Pkg")
-    jl.Pkg.add(name="Wflow", rev="BMI")
-    jl.Pkg.add("BasicModelInterface")
-
-
-def check_wflow_install():
-    """Check if Wflow is installed in the Juliacall environment."""
-    try:
-        jl.seval("using Wflow")
-    except JuliaError as e:
-        if "not found in current path" in str(e):
-            install_wflow()
-        else:
-            raise
-
-
-class WflowBmi(BmiJulia):
-    """Wflow.jl Basic Model Interface."""
-
-    def __init__(self):
-        """Wflow.jl Basic Model Interface."""
-        check_wflow_install()
-
-        m = self.from_name("Wflow.Model", implementation_name="Wflow.BMI")
-        super().__init__(m.model, m.implementation)
-
-
-class WflowJl(WflowJlMixins, LocalModel):
+class WflowJl(WflowJlMixins, ContainerizedModel):
     """Wflow.jl eWaterCycle LocalModel."""
 
-    bmi_class: type[Bmi] = WflowBmi
+    bmi_image: ContainerImage = ContainerImage(
+        "ghcr.io/ewatercycle/wflowjl-remotebmi:0.2.0"
+    )
+    protocol: Literal["grpc", "openapi"] = "openapi"
 
     def get_latlon_grid(self, name: str) -> tuple[np.ndarray, np.ndarray, np.ndarray]:
```

Comment:
Are the methods below still needed?

Reply:
Yep they are, unstructured grid has x,y as lat,lon and start_time = 0 and time units is 's'.